skrifa/src/scale/glyf/glyph.rs

```diff
@@ -48,13 +48,13 @@ pub struct ScalerGlyph<'a> {
 impl<'a> ScalerGlyph<'a> {
     /// Returns the minimum size in bytes required to scale an outline based
     /// on the computed sizes.
-    pub fn required_buffer_size(&self) -> usize {
+    pub fn required_buffer_size(&self, with_hinting: bool) -> usize {
```

Comment:
I would prefer an enum to a bool; seeing `required_buffer_size(true)` isnt' very intelligible